app/views/layouts/application.html.erb

```diff
@@ -37,54 +36,69 @@
     <a class="usa-skipnav" href="#main-content">Skip to main content</a>
 
     <header class="usa-banner">
-      <div class="usa-accordion">
-        <div class="usa-banner__header">
-          <div class="usa-banner__inner">
-            <div class="grid-col-auto">
-              <%= image_tag 'img/us_flag_small.png', class: 'usa-banner__header-flag', alt: 'U.S. flag' %>
-            </div>
-            <div class="grid-col-fill tablet:grid-col-auto">
-              <p class="usa-banner__header-text">An official website of the United States government</p>
-              <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
+      <section class="usa-banner" aria-label="Official government website">
+        <div class="usa-accordion">
+          <div class="usa-banner__header">
+            <div class="usa-banner__inner">
+              <div class="grid-col-auto">
+                <%= image_tag 'us_flag_small.png', class: 'usa-banner__header-flag', alt: 'U.S. flag' %>
+              </div>
+              <div class="grid-col-fill tablet:grid-col-auto">
+                <p class="usa-banner__header-text">An official website of the United States government</p>
+                <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
+              </div>
+              <button class="usa-accordion__button usa-banner__button"
+                aria-expanded="false" aria-controls="gov-banner">
+                <span class="usa-banner__button-text">Here’s how you know</span>
+              </button>
             </div>
-            <button class="usa-accordion__button usa-banner__button"
-              aria-expanded="false" aria-controls="gov-banner">
-              <span class="usa-banner__button-text">Here’s how you know</span>
-            </button>
           </div>
-        </div>
-        <div class="usa-banner__content usa-accordion__content" id="gov-banner" hidden>
-          <div class="grid-row grid-gap-lg">
-            <div class="usa-banner__guidance tablet:grid-col-6">
-              <%= image_tag 'img/icon-dot-gov.svg', alt: 'Dot gov', class: 'usa-banner__icon usa-media-block__img' %>
-              <div class="usa-media-block__body">
-                <p>
-                  <strong>The .gov means it’s official.</strong>
-                  <br>
-                  Federal government websites often end in .gov or .mil. Before sharing sensitive information, make sure you’re on a federal government site.
-                </p>
+          <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+            <%= javascript_tag(nonce: true) do %>
+              document.getElementById('gov-banner').setAttribute('hidden', '');
+            <% end %>
```

Comment:
I'm seeing a CSP error in my console when testing in local development.

For this to work, it may be necessary to configure nonce directives to at least include nonce in `script-src`:

https://github.com/18F/identity-dashboard/blob/1a6a438a8be5f442e478786eaca2ec0bd5480707/config/initializers/content_security_policy.rb#L21

See equivalent IdP code: https://github.com/18F/identity-idp/blob/0621238/config/initializers/content_security_policy.rb#L50

Reply:
maybe i'll just take this out and worry about it another day; i grabbed it from the idp since i was using it as reference for the banner. don't really need to add csp scope to this pr

Reply:
removed in https://github.com/18F/identity-dashboard/commit/20e94541458f876cf2ab980a6a9c511e0bce11dc

Reply:
Oh I see you added back the `hidden` attribute too, so that should solve that flickering.